lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -456,20 +480,36 @@ def delete_comment(request, course_id, comment_id):
     return JsonResponse(prepare_content(comment.to_dict(), course_key))
 
 
+def _vote_or_unvote(request, course_id, obj, value):
+    """
+    Vote or unvote for a thread or a response.
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    course = get_course_with_access(request.user, 'load', course_key)
+    user = cc.User.from_django_user(request.user)
+    if value == 'undo':
```

Comment:
Thanks, that's a good suggestion.  The reason for the current version was that there wouldn't be a useful value for `value` when undoing votes.